common/src/main/java/com/wynntils/core/functions/Function.java

```diff
@@ -10,11 +10,19 @@
 import net.minecraft.client.resources.language.I18n;
 
 public abstract class Function<T> implements Translatable {
+    private final String nameCache;
+    private final String translationNameCache;
+
+    public Function() {
+        String name = this.getClass().getSimpleName().replace("Function", "");
+        this.nameCache = CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_UNDERSCORE, name);
+        this.translationNameCache = CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, name);
```

Comment:
Can't this just be name and translationName

Reply:
I like to use this in constructors. Or are you talking about smth else?

Reply:
oh, i see what you mean. Sure I can rename them.